### DISCUSSION TITLE
Add contributor-only categories/"permissions"

### DISCUSSION THREAD

**Discussion by EliteMasterEric** [community/community#3284] [Discussions] [answered]
It would be useful to be able to restrict certain categories to only allow repository contributors to create discussions in them.

This would be useful in the case of something like an announcements category, where it would be inconvenient or misleading to have other users posting in it.

**Comment by evi-liu** [comment, marked as answer]
Please see this [post](https://github.com/github/feedback/discussions/2937) for more on this request!
